frontend/src/AppFrontend.vue

```diff
@@ -1,33 +1,360 @@
+                  </v-row>
+                </v-list-item-title>
+                <v-list-item-subtitle
+                  v-text="item.description"
+                ></v-list-item-subtitle>
+                <v-list-item-subtitle class="caption">
+                  Updated
+                  <timeago :datetime="item.updatedAt"></timeago>
+                </v-list-item-subtitle>
+              </v-list-item-content>
+            </template>
+          </v-autocomplete>
+        </v-responsive>
+        <v-menu v-if="user" bottom left offset-y class="userMenu">
+          <template #activator="{ on, attrs }">
+            <v-btn
+              icon
+              v-bind="attrs"
+              height="38"
+              width="38"
+              class="ml-3"
+              v-on="on"
+            >
+              <v-avatar color="background" size="38">
+                <v-img v-if="user.avatar" :src="user.avatar" />
+                <v-img
+                  v-else
+                  :src="`https://robohash.org/` + user.id + `.png?size=38x38`"
+                />
+              </v-avatar>
+            </v-btn>
+          </template>
+          <v-list dense class="userMenu" color="background2">
+            <v-list-item>
+              <v-list-item-content class="caption">
+                Signed in as:
+                <strong>{{ user.name }}</strong>
+              </v-list-item-content>
+            </v-list-item>
+            <v-divider></v-divider>
+            <v-list-item
+              v-if="!this.$vuetify.theme.dark"
+              link
+              @click="switchTheme"
+            >
+              <v-list-item-content>Dark mode</v-list-item-content>
+              <v-list-item-icon>
+                <v-icon>mdi-weather-night</v-icon>
+              </v-list-item-icon>
+            </v-list-item>
+            <v-list-item v-else exact @click="switchTheme">
+              <v-list-item-content>Light mode</v-list-item-content>
+              <v-list-item-icon>
+                <v-icon>mdi-white-balance-sunny</v-icon>
+              </v-list-item-icon>
+            </v-list-item>
+            <!-- <v-list-item href="https://speckle.systems/" target="_blank">
+                <v-list-item-content>SpeckleSystems</v-list-item-content>
+              </v-list-item> -->
+            <v-list-item @click="signOut">
+              <v-list-item-content>Sign out</v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </v-container>
     </v-app-bar>
-    <v-navigation-drawer v-model="drawer" app clipped color="grey lighten-4">
-      So you think somthig should be here? 
-    </v-navigation-drawer>
-    <v-content>      
+
+    <v-main :style="background">
       <router-view></router-view>
-    </v-content>
+    </v-main>
   </v-app>
 </template>
 <script>
-import HelloWorld from './components/HelloWorld';
+import userQuery from "./graphql/user.gql"
+import gql from "graphql-tag"
 
 export default {
```

Comment:
Search doesn't seem to be debounced? Not a blocker but to remember this is a self-inflicted DDoS route that should be patched.

Reply:
Not yet implemented, would you like me to add it to this PR or make an issue for it?